website_sale_hide_price/__manifest__.py

```diff
@@ -2,9 +2,9 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "Website Sale Hide Price",
-    "version": "16.0.2.0.0",
+    "version": "17.0.1.0.0",
     "category": "Website",
-    "author": "Tecnativa, " "Odoo Community Association (OCA)",
+    "author": "Tecnativa, Odoo Community Association (OCA), Idealis Consulting",
```

Comment:
You can set yourself as contributor in the readme, but migration works (unless there's a substantial refactor of code) doesn't entitle authorship of the module

```suggestion
    "author": "Tecnativa, Odoo Community Association (OCA)",
```

Reply:
Hello, I made this modification and happen the modification on my previous commit